integreat_cms/api/v3/socialmedia_headers.py

```diff
@@ -0,0 +1,71 @@
+"""
+This module includes functions related to the pages API endpoint.
+"""
+import logging
+from html import unescape
+
+from django.conf import settings
+from django.shortcuts import render
+from django.utils.html import strip_tags
+from django.utils.translation import activate
+from django.utils.translation import gettext as _
+
+from ...cms.models import PageTranslation
+
+logger = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+def socialmedia_headers(request, region_slug, language_slug, path):
+    """
+    Retrieves all children for a single page
+
+    :param request: The request that has been sent to the Django server
+    :type request: ~django.http.HttpRequest
+
+    :param region_slug: Slug defining the region
+    :type region_slug: str
+
+    :param language_slug: Code to identify the desired language
+    :type language_slug: str
+
+    :param path: Should be a page translation slug
+    :type path: str
+
+    :return: HTML meta headers required by social media platforms
+    :rtype: ~django.template.response.TemplateResponse
+    """
+    region = request.region
+    page_translation = PageTranslation.search(region, language_slug, path).first()
+    activate(language_slug)
+
+    if page_translation:
+        excerpt = (
+            unescape(strip_tags(page_translation.content))[:100]
+            if page_translation
+            else None
```

Comment:
`None` does also not make sense here, we'd probably want to return a 404 error in case the requested page does not exist (or is not public)?

Reply:
Yes, good idea.